pkg/apis/configuration/validation/dos.go

```diff
@@ -0,0 +1,180 @@
+package validation
```

Comment:
shall this file belong in `apis/dos/validation` rather than here?

Reply:
moved